--- eSASS/autoconf/setup/esass-init.sh.in
+++ eSASS/autoconf/setup/esass-init.sh.in	(date 1783875267070)
@@ -2,8 +2,9 @@
 # Setup script for eSASS environment
 # This file has been automatically generated at eSASS compile time

-export HEADAS=@HEASOFT@
-source $HEADAS/headas-init.sh
+# Automatic HEADAS setting and HEASoft activation has been patched out of this
+#  Conda-packaged version of eSASS - it will be handled by the HEASoft Conda
+#  package activation scripts.

 export SASS_ROOT=$(realpath $(dirname $BASH_SOURCE)/../)
 export SASS_DIR=$SASS_ROOT


--- eSASS/autoconf/setup/esass-init.csh.in
+++ eSASS/autoconf/setup/esass-init.csh.in	(date 1783956980985)
@@ -2,8 +2,9 @@
 # Setup script for eSASS environment
 # This file has been automatically generated at eSASS compiletime

-setenv HEADAS @HEASOFT@
-source $HEADAS/headas-init.csh
+# Automatic HEADAS setting and HEASoft activation has been patched out of this
+#  Conda-packaged version of eSASS - it will be handled by the HEASoft Conda
+#  package activation scripts.

 setenv SASS_ROOT @prefix@
 #setenv SASS_DIR $SASS_ROOT


--- eSASS/autoconf/setup/esass-init.zsh.in
+++ eSASS/autoconf/setup/esass-init.zsh.in	(date 1783957782537)
@@ -2,8 +2,10 @@
 # Setup script for eSASS environment
 # This file has been automatically generated at eSASS compile time

-export HEADAS=@HEASOFT@
-source $HEADAS/headas-init.sh
+# Automatic HEADAS setting and HEASoft activation has been patched out of this
+#  Conda-packaged version of eSASS - it will be handled by the HEASoft Conda
+#  package activation scripts.
+
 if [[ $SHELL != *"zsh"* ]];
 then
  echo " This script is for ZSH shell, you are running ${SHELL}"
